about-me.html

```diff
@@ -0,0 +1,124 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8"> 
+	<title>Eleanor Stevens</title>
+	<link href="portfolio-stylesheet.css" type="text/css" rel="stylesheet">
+	<link href="normalize.css" type="text/css" rel="stylesheet">  <!-- this means the style is universal across all browsers -->
+	<link href="DINNextLTPro-Regular.ttf" type="text/ttf" rel="stylesheet">
+	<link href="https://github.com/elstevens23" rel="stylesheet">
+	
+</head>
+<body>
+<section class="navBar">
+	<nav>
+		<a href="portfolio.html">HOME</a>
+		<a href="about-me.html">ABOUT ME</a>
+		<a href="web-portfolio.html">PORTFOLIO</a>
+		<a href="contact.html">CONTACT ></a>
+	</nav>
+	
+	<h1 class="pageTitle"> <a href="portfolio.html">Eleanor Stevens</a> </h1>
+	<!-- <h2 class="subheading">WEBSITE DESIGN & DEVELOPMENT</h2> -->
+	<div class="right">
+	</div>
+	
+</section>
+
+<section class="aboutme">
+	<div class="top-aboutme">
+		<img src="img_avatar2.jpg" alt="Woman cartoon avatar">
+		<div class="aboutme-heading">
+			<h1><strong>About Me</strong></h1>
+		<p>Hi, my name is Eleanor and I am training to be 
+		<br>
+		a full-stack developer. I am currently studying 
+		<br>
+		at Mayden Academy, in Bath.</p>
+		</div> 
+	</div>
+	
+	<div class="middle-aboutme">
+		<div class="aboutheading-two">
+			<h1>What I do</h1>
+		</div>	
+		<div class="icons">
```

Comment:
this part looks good in the browser when its "full screen" but when you make the browser screen smaller you have to scroll right to see it all, could change the grid template for smaller screens?